game/CMakeLists.txt

```diff
@@ -67,22 +67,19 @@ set(SOURCES ${SOURCE_FILES} ${HEADER_FILES} ${RESOURCE_FILES})
 # Build main executable
 add_executable(performous ${SUBSYSTEM_WIN32} ${SOURCES} ${SDL2_SOURCES})
 # Libraries
-
-find_package(Boost 1.55 REQUIRED COMPONENTS program_options iostreams system locale)
+find_package(Boost ${NEEDED_BOOST_VERSION} REQUIRED COMPONENTS program_options iostreams system locale)
 target_include_directories(performous PRIVATE ${Boost_INCLUDE_DIRS})
 target_link_libraries(performous PRIVATE ${Boost_LIBRARIES})
-
 add_library(fs STATIC fs.cc)
 target_include_directories(fs PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_link_libraries(fs PRIVATE nlohmann_json::nlohmann_json)
 if (USE_BOOST_FS)
-    find_package(Boost 1.55 REQUIRED COMPONENTS filesystem)
-
+    find_package(Boost ${NEEDED_BOOST_VERSION} REQUIRED COMPONENTS filesystem)
     target_link_libraries(fs PUBLIC Boost::filesystem)
     target_compile_definitions(fs PUBLIC -DUSE_BOOST_FS)
 else()
     find_package(Filesystem REQUIRED)
     target_link_libraries(fs PRIVATE std::filesystem)
-endif()
```

Comment:
This shouldn't have been deleted (`cmake ..` error)

Reply:
Must have been a botched merge from the rebase on master.